test/memspaces/memspace_numa.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2023 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "memspaces/memspace_numa.h"
+#include "base.hpp"
+#include "memspace_helpers.hpp"
+#include "memspace_internal.h"
+
+#include <umf/providers/provider_os_memory.h>
+
+TEST_F(numaNodesTest, createDestroy) {
+    umf_memspace_handle_t hMemspace = nullptr;
+    enum umf_result_t ret = umfMemspaceCreateFromNumaArray(
+        nodeIds.data(), nodeIds.size(), &hMemspace);
+    ASSERT_EQ(ret, UMF_RESULT_SUCCESS);
+    ASSERT_NE(hMemspace, nullptr);
+
+    umfMemspaceDestroy(hMemspace);
+}
+
+TEST_F(numaNodesTest, createNullArray) {
```

Comment:
please rename negative tests eg.:
createNullArray -> create_INVALID_ARRAY
createZeroSize -> create_INVALID_SIZE
etc

please see how it is done in provider_os_memory.cpp

Reply:
I see that we have `TEST_F` and `TEST_P` test cases that use camel case and snake case in its names. What case should we stick to in UMF?

Reply:
In general, I believe we should unify the naming convention. I can see most of the test names use camelCase, so, perhaps, use that one as well.

That said, I think Rafał meant to change words null/zero to `invalid`...? and probably add more cases, as in provider_os_memory test

Reply:
I've marked the negative test cases with `Invalid` in the test name.